quick_build_test.sh

```
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Quick build test to check for compilation errors
+cd /home/daniel/work/puzzgameFlutter
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)